geofileops/util/_geoops_sql.py

```diff
@@ -250,6 +250,8 @@ def isvalid(
 
     # Check the number of invalid files
     nb_invalid_geoms = 0
+    if not output_path.parent.exists():
+        raise ValueError(f"Output directory does not exist: {output_path.parent}")
```

Comment:
```suggestion
```
The check here is can be removed as it is not relevant. The output file has already been written in `_single_layer_vector_operation` (and `_single_layer_vector_operation` already contains the directory check before it starts processing)... so no use checking here. The `if output_path.exists():` check is for another reason here than in all other places in the codebase.